pipeline/post-results-new.groovy

```diff
@@ -0,0 +1,278 @@
+
+                    def latestContent = sharedLib.readFromReleaseFile(
+                        majorVersion, minorVersion
+                    )
+                    println("latestContentBefore: ${latestContent}")
+
+                    if ( latestContent.containsKey(tierLevel) ) {
+                        latestContent[tierLevel] = composeInfo
+                    }
+                    else {
+                        def updateContent = ["${tierLevel}": composeInfo]
+                        latestContent += updateContent
+                    }
+                    println("latestContent: ${latestContent}")
+                    sharedLib.writeToReleaseFile(
+                        majorVersion, minorVersion, latestContent
+                    )
+                }
+            }
+        }
+
+        stage('updateResultsFile'){
+            println("Stage updateResultsFile")
+            if (composeInfo != null){
+                def launchId = metaData['rp_link'].tokenize('/')[-1]
+                println("Fetching rp_launch_details")
+                def rp_launch_details = reportLib.fetchTestItemIdForLaunch(
+                    launchId,
+                    rp_base_link,
+                    rpPreprocDir,
+                    credsRpProc,
+                    metaData,
+                    stageLevel,
+                    run_type
+                )
+                reportLib.writeToResultsFile(
+                    msgMap["artifact"]["version"],
+                    run_type,
+                    tierLevel,
+                    stageLevel,
+                    metaData['results'],
+                    msgMap['run']['url'],
+                    metaData['rp_link'],
+                    rp_launch_details
+                )
+            }
+        }
+
+        stage('sendConsolidatedReport'){
+            println("Stage sendConsolidatedReport")
+            if (msgMap["pipeline"]["final_stage"] && tierLevel == "tier-2") {
+                def rhcsVersion = sharedLib.getRHCSVersionFromArtifactsNvr()
+                majorVersion = rhcsVersion["major_version"]
+                minorVersion = rhcsVersion["minor_version"]
+                minorVersion = "${minorVersion}"
+
+                reportLib.updateConfluencePage(
+                    sharedLib,
+                    majorVersion,
+                    minorVersion,
+                    msgMap["artifact"]["version"],
+                    run_type,
+                    testResults
+                )
+                emailLib.sendConsolidatedEmail(
+                    run_type,
+                    metaData,
+                    majorVersion,
+                    minorVersion,
+                    msgMap["artifact"]["version"]
+                )
+                sharedLib.sendGChatNotification(
+                    run_type, metaData["results"], tierLevel, stageLevel, build_url
+                )
+            }
+        }
+
+        stage('sendUMBFortier0RC'){
```

Comment:
`sendUMBFortier0RC` ??

Reply:
Stage is to send out UMB messages for stable RC builds to be used by OSP interop team
Unable to add skip stage logic,
-  tried using the option when, 
- Also tried manually setting stage status
none of the above worked. 
Will work on this as part of a different PR